spk/i2pd/Makefile

```diff
@@ -0,0 +1,28 @@
+SPK_NAME = i2pd
+SPK_VERS = 2.18.0
+SPK_REV = 0
+SPK_ICON = src/i2pd.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = publicarray
+DESCRIPTION  = The I2P network provides strong privacy protections for communication over the Internet. Many activities that would risk your privacy on the public Internet can be conducted anonymously inside I2P.
+RELOAD_UI = no
+DISPLAY_NAME = i2pd
+STARTABLE = no
+CHANGELOG = Initial release
+
+HOMEPAGE   = https://geti2p.net/
+HELPURL    = https://i2pd.readthedocs.io/en/latest/
```

Comment:
Sorry that documentation is not available specifically for version 2.18.0...

Reply:
Sorry, I don't understand what you mean.